app_spec/test/index.js

```diff
@@ -1,51 +1,85 @@
 const path = require('path')
 const tape = require('tape')
 
-const { Diorama, tapeExecutor, backwardCompatibilityMiddleware } = require('@holochain/diorama')
+const { Orchestrator, tapeExecutor, backwardCompatibilityMiddleware } = require('@holochain/try-o-rama')
+const spawnConductor = require('./spawn_conductors')
 
 process.on('unhandledRejection', error => {
   // Will print "unhandledRejection err is not defined"
   console.error('got unhandledRejection:', error);
 });
 
 const dnaPath = path.join(__dirname, "../dist/app_spec.dna.json")
-const dna = Diorama.dna(dnaPath, 'app-spec')
-const dna2 = Diorama.dna(dnaPath, 'app-spec', {uuid: 'altered-dna'})
+const dna = Orchestrator.dna(dnaPath, 'app-spec')
+const dna2 = Orchestrator.dna(dnaPath, 'app-spec', {uuid: 'altered-dna'})
 
-const dioramaSimple = new Diorama({
+const commonConductorConfig = {
   instances: {
-    alice: dna,
-    bob: dna,
-    carol: dna,
+    app: dna,
+  },
+}
+
+const orchestratorSimple = new Orchestrator({
+  conductors: {
+    alice: commonConductorConfig,
+    bob: commonConductorConfig,
+    carol: commonConductorConfig,
   },
-  bridges: [
-    Diorama.bridge('test-bridge', 'alice', 'bob')
-  ],
   debugLog: false,
   executor: tapeExecutor(require('tape')),
   middleware: backwardCompatibilityMiddleware,
 })
 
-const dioramaMultiDna = new Diorama({
-  instances: {
-    alice: dna,
-    bob: dna2,
+const orchestratorMultiDna = new Orchestrator({
+  conductors: {
+    conductor: {
+      instances: {
+        app1: dna,
+        app2: dna2,
+      },
+      bridges: [
+        Orchestrator.bridge('test-bridge', 'app1', 'app2')
+      ],
+    }
   },
-  bridges: [
-    Diorama.bridge('test-bridge', 'alice', 'bob')
-  ],
   debugLog: false,
   executor: tapeExecutor(require('tape')),
   middleware: backwardCompatibilityMiddleware,
+  callbacksPort: 8888,
 })
 
-require('./regressions')(dioramaSimple.registerScenario)
-require('./test')(dioramaSimple.registerScenario)
-require('./multi-dna')(dioramaMultiDna.registerScenario)
+require('./regressions')(orchestratorSimple.registerScenario)
+require('./test')(orchestratorSimple.registerScenario)
+require('./multi-dna')(orchestratorMultiDna.registerScenario)
 
 const run = async () => {
-  await dioramaSimple.run()
-  await dioramaMultiDna.run()
+  const alice = await spawnConductor('alice', 3000)
+  await orchestratorSimple.registerConductor({name: 'alice', url: 'http://0.0.0.0:3000'})
+  const bob = await spawnConductor('bob', 4000)
+  await orchestratorSimple.registerConductor({name: 'bob', url: 'http://0.0.0.0:4000'})
+  const carol = await spawnConductor('carol', 5000)
+  await orchestratorSimple.registerConductor({name: 'carol', url: 'http://0.0.0.0:5000'})
+
+  const delay = ms => new Promise(resolve => setTimeout(resolve, ms))
+  console.log("Waiting for conductors to settle...")
+  await delay(5000)
```

Comment:
could this come back to bite us? What is this waiting, more specifically?

Reply:
I believe this is mainly about having n3h fully booted and nodes connected. First tests were failing sometimes without this.

Reply:
I wonder if this is still needed. At one point hachiko was not hooked up properly and we were diagnosing the problem. I can't currently run these tests locally so can't confirm. Maybe we try removing it and running this in CI?

I get this error running locally: `error: Test conductor aborted: {"code":-32603,"message":"No space left on device (os error 28)"}`